models/action.json

```diff
@@ -15,11 +19,16 @@
     "result": {
       "type": "object",
       "description": "The result or output of the action"
+    },
+    "serviceInstanceId": {
+      "type": "number",
```

Comment:
It is a shame these foreign keys have to be defined just to mark them as required instead of being able to define the relation as required instead.

Fun fact: the `type`'s for these foreign keys appear to get overridden by the type specified on the id field when the relation is defined.